tests/test_jump.py

```diff
@@ -275,14 +273,15 @@ def test_inputjump_sat_star():
                                             min_sat_radius_extend=2.5,
                                             sat_expand=2)
     assert(num_extended_events == 312)
-    fits.writeto("outgdq2.fits", testcube, overwrite=True)
+    fits.writeto(tmp_path / "outgdq2.fits", testcube, overwrite=True)
 
 
-@pytest.mark.skip("Used for local testing")
-def test_inputjump_sat_star2():
-    testcube = fits.getdata('input_gdq_satstar.fits')
+# TODO: The input data for this is missing, need to find it
+@pytest.mark.xfail(reason="Missing Input Data")
+def test_inputjump_sat_star2(tmp_path, data_path):
+    testcube = fits.getdata(data_path / 'input_gdq_satstar.fits')
```

Comment:
This was added by PR #144, if @mwregan2 could provide the `input_gdq_satstar.fits` data I would be happy to enable this. Otherwise I suggest we remove this test.